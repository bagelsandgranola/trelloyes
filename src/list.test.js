import React from 'react';
import ReactDOM from 'react-dom';
import List from './list';
import renderer from 'react-test-renderer';
import STORE from './store.js';

const lists = STORE.lists;
console.log(lists);

const allCards = STORE.allCards;
console.log(allCards);

const cardIDs = lists[0].cardIds;
console.log(cardIDs);

{STORE.lists.map(list => (
    <List 
    key={list.id} 
    header={list.header} 
    cards={list.cardIds.map(id => (

        allCards[id]
    )

    )}>

    </List>
    ))}

it ('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<List/>, div);
    ReactDOM.unmountComponentAtNode(div);

})

it ('renders the UI as expected', () => {

    const tree = renderer
    .create(
        <List
        key="1"
        header="header"
        cards={lists}
       
        >
        </List>
        )
    .toJSON();
    expect(tree).toMatchSnapshot();

})