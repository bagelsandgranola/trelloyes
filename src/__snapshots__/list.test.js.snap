// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<section
  className="List"
>
  <header
    className="List-header"
  >
    <h2>
       
      header
       
    </h2>
  </header>
  <div
    className="List-cards"
  >
    <div>
      <div
        className="Card"
      >
        <h3>
           
        </h3>
        <p>
           
        </p>
      </div>
      <div
        className="Card"
      >
        <h3>
           
        </h3>
        <p>
           
        </p>
      </div>
    </div>
  </div>
</section>
`;
